packages/components/src/components/Log/Log.test.js

```diff
@@ -16,21 +16,25 @@ import Log from './Log';
 import { renderWithIntl } from '../../utils/test';
 
 it('Log renders default content', () => {
-  const { queryByText } = renderWithIntl(<Log />);
+  const { queryByText } = renderWithIntl(<Log fetchLogs={() => undefined} />);
   expect(queryByText(/No log available/i)).toBeTruthy();
 });
 
-it('Log renders the provided content', () => {
```

Comment:
@AlanGreene The loading state is stopping me from having this and It really should be covered. Tried a few things but nothing worked. Any suggestions on how I can wait for the component to be past the loading state in the test ?

Reply:
You can use `waitForElement` from `react-testing-library` to wait for the expected text. If it times out (5 seconds?) the test fails.

```diff
-expect(queryByText(/testing/i)).toBeTruthy();
+await waitForElement(() => getByText(/testing/i));
```
```